### ISSUE TITLE
Requests to favicon.ico in the #sharp branch return a 500

### ISSUE THREAD

**Issue by jgillman** [jimmynicol/image-resizer#37]
Just a mixup with `request` and `response`. Happens to me all the time :smile:, they both start with `re-`!

Details here: https://github.com/jimmynicol/image-resizer/commit/2bfb2c8cd462812aa1a6027c57bd568b0b8c876e#commitcomment-9755749

**Comment by jimmynicol** [comment]
Fixed in the latest `sharp` branch.

**Comment by jgillman** [comment]
This is not fixed in the latest `sharp` branch. Should be:

```diff
- app.get('/favicon.ico', function (request) {
-   request.sendStatus(404);
+ app.get('/favicon.ico', function (request, response) {
+   response.sendStatus(404);
});
```

Right now `/index.js` errors on line 37 with `TypeError: undefined is not a function`. Browsers heavily cache the favicon so it may be hard to reproduce unless you can change the domain you're testing on.

**Comment by jgillman** [comment]
:+1: Fixed with a844fb4
